public/video-ui/src/services/pandaReqwest.js

```diff
@@ -9,8 +9,8 @@ function poll(reqwestBody, timeout) {
   function makeRequest(resolve, reject) {
     reqwest(reqwestBody)
       .then(response => resolve(response))
-      .fail(err => {
-        if (err === 419) {
+      .catch(err => {
```

Comment:
https://github.com/ded/reqwest

The reqwest github page seems to suggest `fail` is the correct function to use here?